src/components/HomePage.jsx

```diff
@@ -1,22 +1,65 @@
-import React, { Component } from 'react';
-import Image from "./assets/image.jpg";
+import React from 'react';
+import SearchIcon from "./assets/search.png";
+import LocationIcon from "./assets/location.jpg";
 import "bootstrap/dist/css/bootstrap.css";
+import '../styles/home.css'
 import "../index.css";
 
-class HomePage extends Component {
-  render() {
-    return (
-      <React.Fragment>
-        <div className="container">
+
+
+const HomePage = () => {
+  return ( 
+    <React.Fragment>
+        <div className="container text-center">
           <h1>Welcome to Career-Facet</h1>
-          <img src={Image} alt="" />
           <h2>Experience the new way to work.</h2>
           <h3>See jobs we have picked for you.</h3>
-          <button type="button" className="btn btn-lg btn-light">Get Started</button>
+
+          <div className="row justify-content-center">
+            <div className="col-md-6">
+              <div className="input-group mb-3">
+                <div className="input-group-prepend">
+                  <span className="input-group-text">
+                    <img src={LocationIcon} alt="Location" className='searchIcon' style={{ width: "25px", height: "25px" }} />
```

Comment:
You need to move these styles into the style file. Do not mix in-line styling with component-level styling that uses stylesheet files.